examples/test/model/index.html

```diff
@@ -11,13 +11,16 @@
       <a-assets>
```

Comment:
Would be cool to add text labels under each model for the model format. And the scene could use a better background `<a-sky color="#FAFAFA">`